client/state/activity-log/log/schema.js

```diff
@@ -0,0 +1,24 @@
+export const logItemsSchema = {
+	type: 'object',
+	patternProperties: {
+		'^\\d+$': {
+			type: 'array',
+			items: {
+				type: 'object',
+				required: [
+					'name',
+					'ts_utc',
+				],
+				properties: {
+					actor: { type: 'object' },
+					group: { type: 'string' },
+					name: { type: 'string' },
+					object: { type: 'object' },
+					ts_utc: { type: 'integer' },
+				},
+				additionalProperties: false,
```

Comment:
Should this be `false`?

I'm getting a lot of warnings:
<img width="789" alt="screen shot 2017-07-10 at 13 00 18" src="https://user-images.githubusercontent.com/7767559/28017171-d4d809b8-656f-11e7-87e5-3561debefc8f.png">